features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -303,10 +302,10 @@ internal class RumResourceScope(
             )
         }
             .onError {
```

Comment:
I need to do the same thing also for when the resource was stopped due to an error :(